core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.html

```diff
@@ -109,6 +109,7 @@
                                   collection-id="collectionId"
                                   exploration-id="exp.id"
                                   exploration-title="exp.title"
+                                  node-id="nextNodeId"
```

Comment:
Just to check: in story mode, does the "recommended next exploration" show up even if there is no manual specification done in the EndExploration interaction?

If not, it should. In fact, in story mode, we should probably always show just the next chapter + the "return to story" card (and add validation that explorations in stories have no recommendations in the EndExploration).

Reply:
> and add validation that explorations in stories have no recommendations in the EndExploration

This can be added as part of the explorations validations, along with the rest of them. Will add it to the doc.

> does the "recommended next exploration" show up even if there is no manual specification done in the EndExploration interaction

In the screenshot, I hadn't manually changed the EndExploration interaction, but it was still shown. Basically what I am doing here is modify the local `recommendedExplorations` object in conversation skin to store the data of the next exploration in the story, so I could reuse the same html.

Reply:
Done, changed the way this was done. Modified `RecommendationsHandler` in the backend to return the next exploration ID in the story as a 'recommended' exploration, so the same flow is used for going to next story while ignoring any explorations that may been added to EndExploration.